### PR TITLE
Parse pgsql benchmark data and return in readable format for scripts

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -927,6 +927,52 @@ def get_csi_versions():
     return csi_versions
 
 
+def parse_pgsql_logs(data):
+    """
+    Parse the pgsql benchmark data from ripsaw and return
+    the data in list format
+
+    Args:
+      data (str): log data from pgsql bench run
```
Comment (RazTamir) [inline ocs_ci/utility/utils.py]: fix indentation

**Comment by RazTamir** [comment]
Looks good.
Could you please add output example from this function?

**Comment by vasukulkarni** [comment]
my cluster isn't in good state (due to pvc pending issue - new storageclass issue) , I got this data from Tiffany's PR #563 

```
09:35:16 - MainThread - tests.e2e.test_pgsql - INFO - Waiting 120 seconds...
09:37:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc logs -f job.batch/pgbench-benchmark-pgbench-1-dbs-client-1
09:37:17 - MainThread - tests.e2e.test_pgsql - INFO - *******PGBench output log*********
                                                      [{'num_clients': '2', 'latency_avg': '28', 'lat_stddev': '11', 'tps_incl': '66', 'tps_excl': '67'}, {'num_clients': '2', 'latency_avg': '28', 'lat_stddev': '11', 'tps_incl': '66', 'tps_excl': '67'}, {'num_clients': '2', 'latency_avg': '28', 'lat_stddev': '11', 'tps_incl': '66', 'tps_excl': '67'}, {'num_clients': '2', 'latency_avg': '28', 'lat_stddev': '11', 'tps_incl': '66', 'tps_excl': '67'}]
09:37:17 - MainThread - tests.e2e.test_pgsql - INFO - Deleting PG bench benchmark:
09:37:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc delete -f /Users/tunguyen/OCS/ocs-ci/ocs_ci/templates/workloads/pgsql/Benchmark.yaml
PASSED      
```

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -974,6 +974,52 @@ def get_csi_versions():
...
+            eg: ( with only one item in the list)
+            [{'num_clients': '2', 'num_threads': '7', 'latency_avg': '7',
+             'lat_stddev': '0', 'tps_incl': '234', 'tps_excl': '243'}]
+    """
+
+    match = re.findall(
+        r'\[\{\'number_.*?\'number_of_transactions_per_client\':\s+\w+}\]',
+        data
+    )
+    pgsql_data = dict()
```
Comment (clacroix12) [inline ocs_ci/utility/utils.py]: Should this be initialized inside of the for loop?

Comment (vasukulkarni) [reply ocs_ci/utility/utils.py]: yeah will fix that, thx

**Comment by vasukulkarni** [comment]
@RazTamir can I get one approval here?

File: ocs_ci/utility/utils.py [hunk 3]
```diff
@@ -974,6 +974,53 @@ def get_csi_versions():
...
+    Args:
+        data (str): log data from pgsql bench run
+
+    Returns:
+        list_data (list): data digestable by scripts with below
+                        format
+            eg: ( with only one item in the list)
+            [{'num_clients': '2', 'num_threads': '7', 'latency_avg': '7',
+             'lat_stddev': '0', 'tps_incl': '234', 'tps_excl': '243'}]
+    """
```
Comment (petr-balogh) [inline ocs_ci/utility/utils.py]: There should be one free empty line after Returns section

Comment (vasukulkarni) [reply ocs_ci/utility/utils.py]: fixed

**Pull request merged**